### PR TITLE
MIME4J-267 DateTimeFieldLenientImpl should support date with no timezone

### PR DESCRIPTION
**Pull request by chibenwa**

### CODE REVIEW

File: dom/src/test/java/org/apache/james/mime4j/field/LenientDateTimeFieldTest.java [hunk 1]
```diff
@@ -81,4 +81,10 @@ public void testDateWithExtraLeadingWhiteSpace() throws Exception {
         Assert.assertEquals(1175052759000L, f.getDate().getTime());
     }
 
+    @Test
+    public void testDateWhenNoTimezone() throws Exception {
+        DateTimeField f = parse("Date: Fri, 05 Jan 2018 16:18:28 Z");
```
Comment (mbaechler) [inline dom/src/test/java/org/apache/james/mime4j/field/LenientDateTimeFieldTest.java]: Z equals UTC and it's definitely a timezone, it's called Zulu Timezone (no joke)

**Review by mbaechler — CHANGES_REQUESTED** [review]

File: dom/src/test/java/org/apache/james/mime4j/field/LenientDateTimeFieldTest.java (hunk 1 above)
Comment (chibenwa) [reply dom/src/test/java/org/apache/james/mime4j/field/LenientDateTimeFieldTest.java]: Fixed, and in a valid way. CF https://docs.oracle.com/javase/7/docs/api/java/text/SimpleDateFormat.html

Comment (rouazana) [reply dom/src/test/java/org/apache/james/mime4j/field/LenientDateTimeFieldTest.java]: maybe you also need to rename this test then

**Review by strokyl — APPROVED** [review]

**Review by aduprat — APPROVED** [review]

**Review by rouazana — APPROVED** [review]

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**